### PR TITLE
Floppy disk tweaks

### PR DESCRIPTION
**Pull request by comma**
Now all floppies use one dmi instead of.. 5 or so.
Changed disks to use colors and overlays for customization.
image

### CODE REVIEW

**Review by afterthought2 — DISMISSED** [review]

File: code/game/objects/items/weapons/tech_disks.dm [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+	name = "data disk"
+	icon = 'icons/obj/items/device/diskette.dmi'
+	icon_state = ICON_STATE_WORLD
+	w_class = ITEM_SIZE_TINY
+	material = /decl/material/solid/plastic
+	matter = list(/decl/material/solid/metal/steel = MATTER_AMOUNT_TRACE)
+	var/label = "label_blank"
+	
+/obj/item/disk/on_update_icon()
+	. = ..()
```
Comment (MistakeNot4892) [inline code/game/objects/items/weapons/tech_disks.dm:11]: Not much point calling down the tree if you cut overlays right after.

File: code/game/objects/items/weapons/tech_disks.dm [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
+	cut_overlays()
+	var/list/details = list()
+	details += mutable_appearance(icon, "slider", flags = RESET_COLOR)
```
Comment (MistakeNot4892) [inline code/game/objects/items/weapons/tech_disks.dm:14]: Why use MAs for this?

Comment (comma) [reply code/game/objects/items/weapons/tech_disks.dm:14]: whatever proc I rememberd with reset colors flag settable in constructor, not much beyond that.

File: code/game/objects/items/weapons/tech_disks.dm (hunk 1 above)
Comment (comma) [reply code/game/objects/items/weapons/tech_disks.dm:11]: well in case someone needs material alteration stuff down the line.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**